### ISSUE TITLE
OnPrintDialogClose method not fired after print preview closed.

### ISSUE THREAD

**Issue by lute92** [crabbly/Print.js#464]
Hi I am using latest version of print-js. I assigned a call back function in **OnPrintDialogClose** option. It **didn't fired immediately** after the **print preview window was closed** but it only fired after I switched to new browser tab or I press Alt+Tab on Windows OS.

This is my code.
`var afterPreviewClose = function(){
           window.location = "index.html";
 };`

`$("#btnPrint").on("click",function(){
           printJS({
                            printable: 'print-area',
                            type: 'html',
                            onPrintDialogClose: afterPreviewClose, 
                            css: 'styles/styles.css'
           });
  });`

**Comment by lute92** [comment]
This is running sample of above problem.
[Codepen](https://codepen.io/Lute/project/editor/XVrezP#0)

**Comment by JandersonConstantino** [comment]
What is your browser?
Here it works perfectly.

**Comment by lute92** [comment]
I use Chrome lastest version. 
You can see the sample app with the problem here. The first page should redirect to second page after the preview window is closed but it didn't, I implement the onPrintDialogClose option. onPrintDialogClose is not executed immediately after Print Preview window closed but it executes after you switch the current browser tab to another tab or lose focus of the current browser tab.

**Comment by crabbly** [comment]
Hi @lute92 , I just tried you codepen and it works as expected for me. I tested using the latest version of Chrome in MacOS. What OS are you using?

**Comment by lute92** [comment]
Hi @crabbly, I am using the latest version of Chrome in Windows 10 Pro.

**Comment by crabbly** [comment]
I just tested this in Chrome and Firefox on Windows (using [BrowserStack](https://www.browserstack.com/)). Both browsers at their latest version. I was able to replicate the issue with Chrome. With Firefox, the callback works as expected.

That said, looks like the issue is scoped to Chrome on Windows. I honestly have no idea why the chrome api (event) would work differently between two operating systems, this is probably not expected.

I'm not sure this is something we can fix, but let's keep this issue open. I'll look into it in more details whenever I have a chance. Hopefully we will find a solution.
